participants/danielgeorgiev.json

```
@@ -0,0 +1,44 @@
+// vim: ft=jsonc
+// Daniel Georgiev's registration for JSCraftCamp 2025
+{
+  // your real name (required by location host)
+  "realName": {
+    "givenName": "Daniel",
+    "familyName": "Georgiev",
+    "placeFamilyNameFirst": false,
+    "hideFamilyNameOnWebsite": false
+  },
+  // please put in the account name of the PR creator, if you sign up somebody else
+  "githubAccountName": "deinGithubUsername", // <-- Ersetze das mit deinem GitHub-Nutzernamen
+  // company name (optional)
+  "company": "",
+  // either both days or at least one day has to be set to true.
+  "when": {
+    "friday": true,
+    "saturday": true
+  },
+  // if you are willing to take session notes and publish them to github (required)
+  "iCanTakeNotesDuringSessions": true,
+  // your current interests (JS and in general) (required)
+  "tags": ["JavaScript", "Webentwicklung", "HTML", "CSS", "Lernen"],
```

English would be best 😅 

```suggestion
  "tags": ["JavaScript", "Web development", "HTML", "CSS", "Learning"],
```